internal/commands/server/k8s_e2e_test.go

```diff
@@ -408,7 +408,7 @@ func TestHTTPMeshService(t *testing.T) {
 			serviceFive, err := e2e.DeployHTTPMeshService(ctx, cfg)
 			require.NoError(t, err)
 
-			namespace := e2e.Namespace(ctx)
+			namespace := e2e.Namespaces(ctx)[0]
```

Comment:
This change feels a bit dirty. If we want a specific cross-namespace test, I'd just create a new namespace directly in the test that calls for it.

Reply:
Yea that makes sense, I was initially trying to put the services in a second namespaces and so much of that setup logic is abstracted away in the stack creation.